### DISCUSSION TITLE
Welcome to CrowdFunding_DApp Discussions!

### DISCUSSION THREAD

**Discussion by iamankitkathayat** [iamankitkathayat/CrowdFunding_DApp#5] [Announcements]
👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamankitkathayat** [comment]
I guess this discussion needs more contributors

**Comment by ankitBackupKathayat** [comment]
How will you be able to filter out the collaborators?

**Reply by iamankitkathayat** [reply]
I will have a basic search operations about the quality of their repos.
